src/API/Recruit.Api/Queries/GetApplicationReviewQuery.cs

```diff
@@ -0,0 +1,85 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Esfa.Recruit.Vacancies.Client.Domain.Repositories;
+using MediatR;
+using SFA.DAS.Recruit.Api.Helpers;
+using SFA.DAS.Recruit.Api.Models;
+
+namespace SFA.DAS.Recruit.Api.Queries;
+
+public record GetApplicationReviewQuery(long VacancyReference, Guid CandidateId) : IRequest<GetApplicationReviewResponse>
+{
+    public long VacancyReference { get; set; } = VacancyReference;
+    public Guid CandidateId { get; set; } = CandidateId;
+}
+
+public class GetApplicationReviewQueryHandler(IApplicationReviewRepository applicationReviewRepository) : IRequestHandler<GetApplicationReviewQuery, GetApplicationReviewResponse>
+{
+    public async Task<GetApplicationReviewResponse> Handle(GetApplicationReviewQuery request, CancellationToken cancellationToken)
+    {
+        var validationErrors = ValidateRequest(request);
+
+        if (validationErrors.Count != 0)
+        {
+            return new GetApplicationReviewResponse
+            {
+                ResultCode = ResponseCode.InvalidRequest,
+                ValidationErrors = validationErrors.Cast<object>().ToList()
+            };
+        }
+
+        var applicationReview = await applicationReviewRepository.GetAsync(request.VacancyReference, request.CandidateId);
```

Comment:
you might want to check that this is only bringing back successful applications. Its possible for a candidate to withdraw and re apply